index/server/Dockerfile

```
@@ -38,4 +38,4 @@ USER 1001
 
 EXPOSE 8080
 
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT ["bash /entrypoint.sh"]
```

> Also, either:
> - Add `RUN chmod +x /entrypoint.sh` after https://github.com/thepetk/devfile-registry-support/blob/21b3c798866953932547c0b60a035e26dde6f325/index/server/Dockerfile#L13
> **or**
> - Add `bash` to https://github.com/thepetk/devfile-registry-support/blob/21b3c798866953932547c0b60a035e26dde6f325/index/server/Dockerfile#L41

My mistake, test is failing because of my last suggestion. Revert this and do the first one.
```suggestion
ENTRYPOINT ["/entrypoint.sh"]
```

---

Yes I was investigating now :D Thanks for the help!

---

Done :)